lib/src/10_auth/auth_screen.dart

```diff
@@ -128,10 +146,45 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
     );
   }
 
-  Widget _buildCreateAccountScreen(BuildContext context) {
-    final TextEditingController emailController =
-        new TextEditingController(text: this._email);
+  Widget _buildTextBox(
+      TextEditingController controllerName,
+      FocusNode currentNode,
+      FocusNode nextNode,
+      String fieldName,
+      bool autoFocus,
+      Future<Widget> Function() submit) {
+    handleKey(RawKeyEvent key) {
+      if (key is RawKeyDownEvent) {
+        if (key.physicalKey.debugName == 'Enter') {
+          submit();
+        } else {
+          {
+            setState(() {
+              this._error = "";
+            });
+          }
+        }
+      }
+      if ((key.logicalKey.debugName == 'Tab') && (nextNode != null)) {
+        nextNode.requestFocus();
+      }
+    }
 
+    return RawKeyboardListener(
+        focusNode: FocusNode(),
```

Comment:
Is it necessary to assign it?

Reply:
It appears that focusNode is required:

Constructors
RawKeyboardListener({Key key, @required FocusNode focusNode, bool autofocus: false, ValueChanged<RawKeyEvent> onKey, @required Widget child})